designer/package.json

```diff
@@ -6,6 +6,10 @@
   "engines": {
     "node": ">=12"
   },
+  "browserslist": [
```

Comment:
There's a `.browserslistrc` in the root folder of the repo already which is used by babel-preset-env; if switching to this, pls delete it

Reply:
ok i will remove this section